pkg/asset/installconfig/openstack/validation/platform.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/openshift/installer/pkg/types/openstack"
 )
 
+const (
+	// MTU of 1280 is the minimum allowed for IPv6
+	// https://issues.redhat.com/browse/OCPBUGS-2921
+	minimumMTU = 1280
```

Comment:
this should be 1280 + 100, which is OVN-K overhead

https://github.com/openshift/installer/blob/0810957e2ddd9d7fd7815b9d0d07012fe7a7a1de/docs/user/openstack/deploy_dual_stack_cluster.md?plain=1#L15